backend/app/sql/summary_stats_pop_totals.sql

```diff
@@ -0,0 +1,40 @@
+DROP FUNCTION IF EXISTS get_summary_stats_pop_totals;
+CREATE OR REPLACE FUNCTION get_summary_stats_pop_totals(gerrydb_table TEXT)
+RETURNS TABLE (
+    other_pop BIGINT,
+    asian_pop BIGINT,
+    amin_pop BIGINT,
+    nhpi_pop BIGINT,
+    black_pop BIGINT,
+    white_pop BIGINT
```

Comment:
H: Even though we should remove this function, we'll need to add `two_or_more_races_pop` to `get_summary_p1_totals`.